core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -86,6 +92,9 @@
       <span class="oppia-exploration-open-threads-count">
         <[getOpenThreadsCount()]>
       </span>
+      <span class="feedback-open-thread-static-text">
+        open
```

Comment:
I think it may be okay to just combine this span with the count, if you're planning on putting all the text on one line.

Reply:
Yeah that's how I did that.